content/mods/C-multitasking/topics/async-rt/slides.md

```diff
@@ -45,7 +45,7 @@ layout: default
 /// Set up a tokio Runtime and spawn the Future returned by `main`
 #[tokio::main]
 async fn main() {
-    do_stuff.await();
+    do_stuff.await;
```

Comment:
```suggestion
    do_stuff().await;
```

Reply:
Thanks again! I think it'd be more clear and recognizable if we do keep the parentheses, just move them to the right spot

Reply:
Thanks, looks good.